ui/app/src/components/LogConsoleGrid/LogConsoleGridUI.tsx

```diff
@@ -57,11 +58,13 @@ export default function LogConsoleGridUI(props: LogConsoleGridUIProps) {
                 <FormattedMessage id="words.thread" defaultMessage="Thread" />
               </Typography>
             </GlobalAppGridCell>
-            <GlobalAppGridCell align="left" className="width20">
-              <Typography variant="subtitle2">
-                <FormattedMessage id="words.site" defaultMessage="Site" />
-              </Typography>
-            </GlobalAppGridCell>
+            {showSite && (
+              <GlobalAppGridCell align="left" className="width30">
```

Comment:
The idea is to limit to a max of 30 chars; but we don't want to increase the width of this one. By adjusting this one, any spare width should go to the message column.